contracts/Commitment/PublicCommitment.sol

```diff
@@ -13,6 +14,13 @@ contract PublicCommitment is CommitmentBase {
     {
         // enable Neumark trading in token controller
         neumark.enableTransfer(true);
+        RoleBasedAccessControl(neumark.accessPolicy()).setUserRole(
+            RoleBasedAccessControl.EVERYONE,
+            ROLE_NEUMARK_TRADER,
+            neumark,
+            RoleBasedAccessControl.TriState.Allow
+        );
+
```

Comment:
This would require PublicCommitment to have ROLE_ACCESS_CONTROLLER. I don't like that.